core/src/main/kotlin/com/malinskiy/marathon/report/allure/AllureReporter.kt

```diff
@@ -64,7 +64,7 @@ class AllureReporter(val configuration: Configuration, private val outputDirecto
 
     private fun createTestResult(uuid: String, device: DeviceInfo, testResult: TestResult): io.qameta.allure.model.TestResult {
         val test = testResult.test
-        val fullName = test.toSimpleSafeTestName()
```

Comment:
This change is not related to the test filter, please remove :)

Reply:
Thanks!